code-studio/copy-assets.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+/** @file Build script for copying unpreprocessed assets from the code-studio
+ * package and its dependencies over into the build output directory. */
+'use strict';
+
+var build_commands = require('../shared/js/build-commands');
+
+/** @const {string} */
+var BUILD_PATH = './build/assets/';
```

Comment:
Since they are copied into dashboard/public and also have a precompiled asset rule, they get served in two places:

1. Unprocessed, unfingerprinted at [domain]/code-studio/assets/
2. Preprocessed and fingerprinted at [domain]/assets/[fingerprinted filename]

Option 1 should always be the same, and can be hard-coded as a root-relative URL if needed (I do that in a few places for video.js), but isn't effectively cachebusted, so it's a poor choice for anything we expect to change ourselves.

Option 2 currently requires you to use a Rails asset helper to get the correct URL, but will be cachebusted appropriately because of that.